cfn-templates/data-exports-aggregation.yaml

```diff
@@ -0,0 +1,749 @@
+              - Effect: Allow
+                Action:
+                  - s3:ReplicateObject
+                  - s3:ReplicateDelete
+                  - s3:ReplicateTags
+                  - s3:GetObjectVersionTagging
+                Resource:
+                  Fn::Sub: "arn:${AWS::Partition}:s3:::${ResourcePrefix}-${DestinationAccountId}-shared/cur2/${AWS::AccountId}/*"
+
+####
+# Local CUR2
+####
+
+  ## Deploy CUR2 natively via CFN resource if we are in us-east-1
+  LocalCUR2InSource:
+    Type: AWS::BCMDataExports::Export
+    Condition: DeployCUR2ViaCFNInSource
+    DependsOn:
+      - SourceS3BucketPolicy
+    Properties:
+      Export:
+        DataQuery:
+          QueryStatement: !FindInMap [DataExports, CUR2, DefaultQuery]
+          TableConfigurations:
+            COST_AND_USAGE_REPORT:
+              TIME_GRANULARITY: "HOURLY"
+              INCLUDE_RESOURCES: "TRUE"
+              INCLUDE_MANUAL_DISCOUNT_COMPATIBILITY: "FALSE"
+              INCLUDE_SPLIT_COST_ALLOCATION_DATA: "FALSE"
+        Description: "CUR 2.0 export for aggregation in CID"
+        DestinationConfigurations: 
+          S3Destination:
+            S3Bucket: !If [ IsDestinationAccount, !Ref DestinationS3, !Ref SourceS3 ]
+            S3Prefix: !Sub "cur2/${AWS::AccountId}"
+            S3Region: !Ref AWS::Region
+            S3OutputConfigurations:
+              Overwrite: "OVERWRITE_REPORT"
+              Format: "PARQUET"
+              Compression: "PARQUET"
+              OutputType: "CUSTOM"
+        Name: !Ref ResourcePrefix
+        RefreshCadence:
+          Frequency: "SYNCHRONOUS"
+
+  LocalCUR2InDestination:
+    Type: AWS::BCMDataExports::Export
+    Condition: DeployCUR2ViaCFNInDestination
+    DependsOn:
+      - DestinationS3BucketPolicy # Conditional DependsOn is not supported, so we need 2 resources
+    Properties:
+      Export:
+        DataQuery:
+          QueryStatement: !FindInMap [DataExports, CUR2, DefaultQuery]
+          TableConfigurations:
+            COST_AND_USAGE_REPORT:
+              TIME_GRANULARITY: "HOURLY"
+              INCLUDE_RESOURCES: "TRUE"
+              INCLUDE_MANUAL_DISCOUNT_COMPATIBILITY: "FALSE"
+              INCLUDE_SPLIT_COST_ALLOCATION_DATA: "FALSE"
+        Description: "CUR 2.0 export for aggregation in CID"
+        DestinationConfigurations: 
+          S3Destination:
+            S3Bucket: !If [ IsDestinationAccount, !Ref DestinationS3, !Ref SourceS3 ]
+            S3Prefix: !Sub "cur2/${AWS::AccountId}"
+            S3Region: !Ref AWS::Region
+            S3OutputConfigurations:
+              Overwrite: "OVERWRITE_REPORT"
+              Format: "PARQUET"
+              Compression: "PARQUET"
+              OutputType: "CUSTOM"
+        Name: !Ref ResourcePrefix
+        RefreshCadence:
+          Frequency: "SYNCHRONOUS"
+
+  # Deploy CUR2 via lambda due to missing cfn resource definition
+  # AWS::BCMDataExports::Export outside us-east-1
+  CUR2inUSEAST1:
```

Comment:
Probably need to change this from CUR2inUSEAST1 to CUR2viaCustomResource or something similar as there is also cn-* region.

Reply:
makes sense - will fix